version/version.go

```diff
@@ -22,14 +22,15 @@ import (
 
 const (
 	// OperatorVersionString is the version string of this operator binary.
-	OperatorVersionString = "1.3.0"
+	OperatorVersionString = "1.4.0"
 )
 
 var (
 	// SupportedVersions is a list of chart versions supported by this version of the operator.
 	// It must be synced with the versions.yaml file.
 	SupportedVersions = []string{
 		"1.3.0",
+		"1.4.0",
```

Comment:
Also 1.3.1. Actually this is one of the places that the script that creates a new branch should update.